modules/social_features/social_group/config/install/core.entity_view_display.group.closed_group.small_teaser.yml

```diff
@@ -28,7 +38,7 @@ hidden:
   created: true
   field_group_address: true
   field_group_description: true
-  field_group_image: true
   field_group_location: true
   search_api_excerpt: true
+  social_tagging: true
```

Comment:
@OH17L Do we need to add this, since `social_tagging` is optional?

Reply:
@Kingdutch I see that you did it [here](https://github.com/goalgorilla/open_social/blame/feature/3111959-extend-content-block/modules/social_features/social_group/config/install/core.entity_view_display.group.closed_group.small_teaser.yml#L43). Can you explain why?

Reply:
@OH17L I suspect that was an error in that commit so it can be removed as Taras says

Reply:
Fixed